cachito/web/migrations/versions/cdf17fad3edb_request_env_vars.py

```diff
@@ -23,7 +23,7 @@ def upgrade():
         sa.Column("name", sa.String(), nullable=False),
         sa.Column("value", sa.String(), nullable=False),
         sa.PrimaryKeyConstraint("id"),
-        sa.UniqueConstraint("name", "value"),
+        sa.UniqueConstraint("name", "value", name="uq_environmet_variable_name_value"),
```

Comment:
@lcarva do you mind using the value that Postgres generates by default since we've been using that for other constraints and it'd be nice to be consistent.

Reply:
Sure. What would that be here?

Reply:
```suggestion
        sa.UniqueConstraint("name", "value", name="environment_variable_name_value_key"),
```

Reply:
Done!